packages/shared/src/components/squads/SquadEmptyScreen.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactElement, useContext } from 'react';
+import SettingsContext, { ThemeMode } from '../../contexts/SettingsContext';
+import useMedia from '../../hooks/useMedia';
+import { cloudinary } from '../../lib/image';
+import { tablet } from '../../styles/media';
+import { Image } from '../image/Image';
+import { FlexCentered } from '../utilities';
+
+function SquadEmptyScreen(): ReactElement {
+  const settings = useContext(SettingsContext);
+  const isMobile = !useMedia([tablet.replace('@media ', '')], [true], false);
```

Comment:
Prefer the sidebarRendered here as it wrap this under the hood.
Just easier down the line when changes are needed.